src/Microsoft.Sbom.Api/Executors/IJsonSerializationStrategy.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.Entities;
+using Microsoft.Sbom.Extensions;
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+public interface IJsonSerializationStrategy
+{
+    public void AddToFilesSupportingConfig(ref IList<ISbomConfig> elementsSupportingConfigs, ISbomConfig config);
```

Comment:
What's the motivation for explicitly including `ref` when passing the `IList<ISbomConfig>` in these methods? Isn't `IList` already a reference type?

Reply:
IList isn't a ref type by List is, which is what all the implementations use, so this is redundant. I removed all the ref keywords.